virt/install/uninstalling-virt.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="uninstalling-virt"]
+include::_attributes/common-attributes.adoc[]
+= Uninstalling {VirtProductName}
+:context: uninstalling-virt
+
+toc::[]
+
+You can uninstall {VirtProductName} by using the {product-title} xref:../../web_console/web-console.adoc#web-console-overview_web-console[web console] or the command line interface (xref:../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[CLI]).
+
+[id='uninstalling-virt-web-console_{context}']
+== Uninstalling {VirtProductName} using the web console
+
+You uninstall {VirtProductName} by deleting the {VirtProductName} Operator from the cluster and then deleting the `openshift-cnv` namespace.
+
+include::modules/olm-deleting-operators-from-a-cluster-using-web-console.adoc[leveloffset=+2]
```

Comment:
In our case we still recommend to delete the HyperConverged custom resource (CR) first,
for simpler operators this can be automatically done on operator removal but this is not our case.

Reply:
Won't the HCO reconcile process be an issue if the user tries to delete the HyperConverged CR first? Same question as in your previous comment.